adafruit_rgb_display/rgb.py

```diff
@@ -71,74 +82,90 @@ def _decode_pixel(self, data):
         """Decode bytes into a pixel color."""
         return color565(*struct.unpack(self._DECODE_PIXEL, data))
 
-    def pixel(self, x, y, color=None):
+    def pixel(self, xpos, ypos, color=None):
```

Comment:
I think x and y are perfectly fine for pixel position.

Reply:
reverted to x , y